src/go/k8s/Makefile

```diff
@@ -230,7 +230,7 @@ lint: golangci-lint
 
 .PHONY: lint-fix
 lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
-	$(GOLANGCI_LINT) run --fix
+	$(GOLANGCI_LINT) run --fix --timeout 30m
```

Comment:
Oh we need to remove this or replace it with task file as this is very much out of sync with what CI runs. Not to be fixed in this PR. I can fix it because I missed it when I changed up the linting. Sorry 😓